docs/sphinx/examples/python/providers/iqm.py

```diff
@@ -0,0 +1,58 @@
+import cudaq
+
+# You only have to set the target once! No need to redefine it
+# for every execution call on your kernel.
+# To use different targets in the same file, you must update
+# it via another call to `cudaq.set_target()`
+cudaq.set_target("iqm",
+                 url="http://localhost/cocos",
+                 **{"qpu-architecture": "Adonis"})
+
+# Adonis QPU architecture:
+#       QB1
+#        |
+# QB2 - QB3 - QB4
+#        |
+#       QB5
+
+# Create the kernel we'd like to execute on IQM.
+kernel = cudaq.make_kernel()
+qubits = kernel.qalloc(5)
+kernel.h(qubits[2])  # QB3
```

Comment:
Left over comment? 
```suggestion
kernel.h(qubits[2])
```

Reply:
Qubits indexes start with 1 on the IQM Server-side, so this is more of a note that from the qpu architecture point of view this is "QB3", even if it has index 2 in the code.